### DISCUSSION TITLE
Unable to add your repository

### DISCUSSION THREAD

**Discussion by pavankalyan767** [borgbase/vorta#1883] [FAQ]
I created an ssh key using borgbase and created a repository using borgbase then i tried to add that repository in vorta but i am not able to add the repository it says unable to add your repository I'll just attach the screenshot also please look into it and help me solve this 
![vortanotworking](https://github.com/borgbase/vorta/assets/124815665/c2e65874-58e6-4333-ad6f-99623a577c4a)

**Comment by m3nu** [comment]
- Have you also assigned the new key to the new repo? Just adding it to the account isn't enough.
- If you changed the default key name, you also need to select it in the SSH Key dropdown in Vorta.

**Comment by m3nu** [comment]
More here: https://docs.borgbase.com/faq/#all-connections-to-a-borgbase-repo-fail-with-an-error-immediately

**Comment by pavankalyan767** [comment]
@m3nu  I was able to set it up successfully after going through the documentation 
Thank you
